src/simplewallet/simplewallet.cpp

```diff
@@ -986,12 +997,14 @@ bool simple_wallet::print_fee_info(const std::vector<std::string> &args/* = std:
 
 bool simple_wallet::prepare_multisig(const std::vector<std::string> &args)
 {
+  CHECK_MULTISIG_ENABLED();
   prepare_multisig_main(args, false);
```

Comment:
Well, if there is a `main` method in addition to the direct method for interactive use that one will already check for multisig enabled, so strictly speaking a number of such checks are superfluous, but maybe it's a good idea to have them nevertheless so it's crystal clear there is a check and everybody reading the code sees this as early as possible.